### PR TITLE
feat(admin site): add table of coordinators

### LINKED ISSUE #377: Admin site: show table of coordinators
Create a table of coordinators (similar to #344 fixed by #351) with the following columns:
- ID (from `personal_info`)
- Full name (from `personal_info`)
- Valid (from `is_validated`: green tick if `True`, **red cross** if `False`)
- Admin (from `is_admin`): green tick when `True`, **blank** if `False`
- Skills (from `additional_skills_comment`)
- Project status
- Situational status
- Status last changed (from `status_since`)
- Communication language(s) (from `communication_language_mode`)
- Mentor (default string representation of another `Coordinator` that is mentoring this coordinator; hyperlink to their profile)
- Comment

Filtering should be possible by:
- Valid
- Admin
- Project status
- Situational status
- Communication language(s)
- Mentor

Text search should work for
- ID (both of coordinator and their mentor)
- Full name (both of coordinator and their mentor)
- Skills
- Comment

**UPD** Additional requests after review by owners:
- set maximum column width
- replace default "[by] is admin" filter with something more readable like "[by] admin status"

### PR DESCRIPTION
**Pull request by Ross1832**
closes #377

### CODE REVIEW

File: api/admin/coordinator.py [hunk 1]
```diff
@@ -60,11 +61,54 @@ class CoordinatorLogEventsInline(
...
+
+    ordering = ["personal_info_id"]
+
+    list_filter = (
+        "is_validated",
+        "is_admin",
+        "project_status",
+        "situational_status",
+        "personal_info__communication_language_mode",
+        "mentor",
```
Comment (lemontree210) [inline api/admin/coordinator.py:102]:
```suggestion
```

File: api/admin/coordinator.py [hunk 2]
```diff
@@ -60,11 +61,54 @@ class CoordinatorLogEventsInline(
...
+    )
+
+    search_fields = (
+        "personal_info__id",
+        "personal_info__first_name",
+        "personal_info__last_name",
+        "additional_skills_comment",
+        "comment",
+    )
```
Comment (lemontree210) [inline api/admin/coordinator.py:111]: I guess that here is where we should add something like `mentor__personal_info__id`, `mentor__personal_info__first_name`, `mentor__personal_info__last_name`

**Review by lemontree210 — CHANGES_REQUESTED** [review]
Thank you for the PR! 😃

**Review by lemontree210 — COMMENTED** [review]
One more thing that I suggest is to replace underscores with spaces in `project_status` and `situational_status` (both in heading and in the values of these attributes in the cells). This way the columns will not be too wide.

**Comment by lemontree210** [comment]
Additional requests after review by owners:
- set maximum column width
- replace default "[by] is admin" filter with something more readable like "[by] admin status"

File: api/admin/coordinator.py [hunk 3]
```diff
@@ -60,11 +61,54 @@ class CoordinatorLogEventsInline(
...
-        "__str__",
+        "get_personal_info_id",
+        "get_personal_info_full_name",
+        "get_is_validated",
+        "get_is_admin",
+        "get_additional_skills_comment",
+        "get_project_status",
+        "situational_status",
+        "get_status_since",
+        "get_communication_language_mode",
```
[the remark this answers is not in the document]
Comment (Ross1832) [reply api/admin/coordinator.py]: done

File: api/admin/coordinator.py (hunk 2 above)
Comment (Ross1832) [reply api/admin/coordinator.py]: done

File: api/admin/coordinator.py [hunk 4]
```diff
@@ -60,11 +61,54 @@ class CoordinatorLogEventsInline(
...
+        "mentor",
+        "get_comment",
         "active_groups_count",
```
[the remark this answers is not in the document]
Comment (Ross1832) [reply api/admin/coordinator.py]: Do I need to remove   "active_groups_count" from the code ?

File: README.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ Can be found at `/docs/swagger` or `/docs/redoc` after deployment (or when runni
 3. Run `poetry install` in the project directory to [install dependencies](https://python-poetry.org/docs/basic-usage/#installing-dependencies).
 4. Activate virtual environment: `poetry shell`.
 5. Install pre-commit hooks: `pre-commit install`. They will run on files being committed. `black` and `isort` will fix the issues automatically. To check all code, run `pre-commit run -a`.
-6. Copy [the sample settings file](django_webapps/settings_sample.py) to `settings.py`. For local development, you don't need to make any changes to the contents of the file.
+6. Copy [the sample settings file](django_webapps/settings.py) to `settings.py`. For local development, you don't need to make any changes to the contents of the file.
```
[the remark this answers is not in the document]
Comment (Ross1832) [reply README.md:18]: It likely changed automatically, as I didnt make any modifications to the README.md file.

[the remark this answers is not in the document]
Comment (lemontree210) [reply README.md:18]: These changes are still proposed by you, please revert :)

**Review by lemontree210 — COMMENTED** [review]
Thank you for the improvements! Please note that there are some [additional improvements requested by the owners](https://github.com/samanthasgroup/django-webapps/pull/393#issuecomment-1891638721)

**Review by lemontree210 — APPROVED** [review]

**Pull request merged**